src/Parsers/IAST.h

```diff
@@ -240,9 +241,12 @@ class IAST : public std::enable_shared_from_this<IAST>, public TypePromotion<IAS
     }
 
     // A simple way to add some user-readable context to an error message.
-    std::string formatForErrorMessage() const;
-    template <typename AstArray>
-    static std::string formatForErrorMessage(const AstArray & array);
+    String formatWithHiddenSecrets(size_t max_length = 0, bool one_line = true, bool hilite = false) const;
```

Comment:
Maybe we can call this `formatWithSecretsHidden`, because `formatWithHiddenSecrets` gives the impression that it is with some secrets, but as I understand we remove/hide secrets in this function. or if you have any other suggestions for name? what do you think?

Reply:
I also though about that name. Yeah, I'll rename it.

Reply:
done